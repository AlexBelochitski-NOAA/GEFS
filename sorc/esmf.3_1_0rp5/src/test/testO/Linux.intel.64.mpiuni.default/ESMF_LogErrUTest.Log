20190610 112231.953073 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112231.953183 INFO     PET0  Beginning Test, file ESMF_LogErrUTest.F90, line          86
20190610 112231.953192 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112231.953351 INFO     PET0  PASS  Open Log Test, ESMF_LogErrUTest.F90, line          95
20190610 112231.953493 INFO     PET0  PASS  Open ESMF_LOG_SINGLE Log Test, ESMF_LogErrUTest.F90, line         105
20190610 112231.953525 INFO     PET0  PASS  Write to Single Log Test, ESMF_LogErrUTest.F90, line         116
20190610 112231.956054 INFO     PET0  PASS  Open ESMF_LOG_NONE Log of opened file Test, ESMF_LogErrUTest.F90, line         126
20190610 112231.956067 INFO     PET0  PASS  Open ESMF_LOG_NONE Log Test, ESMF_LogErrUTest.F90, line         136
20190610 112231.956078 INFO     PET0  PASS  Use of separate log Test, ESMF_LogErrUTest.F90, line         147
20190610 112231.956228 INFO     PET0  PASS  Close Log Test, ESMF_LogErrUTest.F90, line         156
20190610 112231.956262 INFO     PET0  Number of failed tests:           0
20190610 112231.956277 INFO     PET0  Ending Test, file ESMF_LogErrUTest.F90, line         520
