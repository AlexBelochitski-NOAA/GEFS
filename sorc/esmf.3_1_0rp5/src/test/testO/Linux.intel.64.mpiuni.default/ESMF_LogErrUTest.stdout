 Starting LogErr Tests
  Beginning Test, file ESMF_LogErrUTest.F90, line          86
  NUMBER_OF_PROCESSORS           1
  PASS  Open Log Test, ESMF_LogErrUTest.F90, line          95
  rc =            0
  PASS  Open ESMF_LOG_SINGLE Log Test, ESMF_LogErrUTest.F90, line         105
  rc =            0
  PASS  Write to Single Log Test, ESMF_LogErrUTest.F90, line         116
  rc =            0
 This ESMF_Log is already open with file 'Single_Log_File'
  PASS  Open ESMF_LOG_NONE Log of opened file Test, ESMF_LogErrUTest.F90, line  
        126
  rc =           -1
  PASS  Open ESMF_LOG_NONE Log Test, ESMF_LogErrUTest.F90, line         136
  rc =            0
  PASS  Use of separate log Test, ESMF_LogErrUTest.F90, line         147
  rc =            0
  PASS  Close Log Test, ESMF_LogErrUTest.F90, line         156
  rc =            0
  Number of failed tests:           0
  Ending Test, file ESMF_LogErrUTest.F90, line         520
